hugegraph-core/src/main/java/com/baidu/hugegraph/util/LZ4Util.java

```diff
@@ -36,7 +36,7 @@
     public static BytesBuffer compress(byte[] bytes, int blockSize) {
         LZ4Factory factory = LZ4Factory.fastestInstance();
         LZ4Compressor compressor = factory.fastCompressor();
-        BytesBuffer buf = new BytesBuffer(bytes.length / 8);
+        BytesBuffer buf = new BytesBuffer(bytes.length);
```

Comment:
add a const var buf_ratio, let init_size=bytes.length/buf_ratio, also update decompress to bytes.length*buf_ratio